source/assets/public/grid/index.css

```diff
@@ -0,0 +1,77 @@
+/* Basic CSS grid */
+
+/* margin right - negative right margin to allow for last col's gutter */
+.grid {
+  margin-right: -4rem;
```

Comment:
I would kind of rather use flexbox or even css-grid (since this is a conference for programmers I'm not too concerned about IE) but this is fine, the site's not going to get too much bigger